PicPi/Event.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  Photo.swift
+//  PicPi
+//
+//  Created by Erick Sanchez on 12/23/20.
+//
+
+import Foundation
+
+/// One to many communication of a value's updates.
```

Comment:
Nice!

Reply:
Yeee, totally ripped this from my team lol but nothing secretive